llvm/lib/Transforms/IPO/RepoPruning.cpp

```diff
@@ -297,12 +297,12 @@ static bool eliminateUnreferencedAndPrunedGOs(Module &M) {
 }
 
 static llvm::Value *createDeclararion(Module &M, llvm::Type *Ty,
```

Comment:
typo I think this should be "createDeclaration".

Reply:
Fixed, thanks.